### PR TITLE
More buttons and cards

### PR DESCRIPTION
**Pull request by Shivani-1524**
New components to be reviewed:
1. star icon toggle button 
2. New button styles
3. Transparent background card
4. card with badges

### CODE REVIEW

File: Components/Button/button.css [hunk 1]
```diff
@@ -32,6 +32,46 @@
...
+  border-radius: 3px;
+  color: var(--dark-grey);
+}
+.subtle-btn:hover {
+  border: 1px solid var(--violet-color);
+}
+
+.link-btn {
+  height: min-content;
+  padding: 0px;
```
Comment (eakansh125) [inline Components/Button/button.css]: A recommendation to use just 0 instead of 0px/0rem. It's a better practice!

**Review by eakansh125 — COMMENTED** [review]
Also another suggestion to use rem or % in place of px if possible.
Apart from that, components look good to merge to me!

File: Components/Card/card.css [hunk 2]
```diff
@@ -11,6 +11,12 @@
...
 
+.no-bg-color {
+  background-color: transparent;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+  transition: all 0.3s cubic-bezier(0.25, 0.8, 0.25, 1);
+}
+
 .hor-card {
   width: 100%;
   min-width: 350px;
```
Comment (manish-GH) [inline Components/Card/card.css:22]: You could use rem here

**Review by shangkaul — APPROVED** [review]
The code for the listed components looks good!

The horizontal card documentation seems a bit messed up you can address this in a later commit, looks good to merge for now!
Screenshot 2022-02-13 at 7 00 30 PM

File: Components/Card/card.css [hunk 3]
```diff
@@ -44,13 +50,18 @@
   grid-template-rows: 250px auto auto;
 }
 .ver-lg-card {
+  width: 100%;
   max-width: 350px;
```
Comment (manish-GH) [inline Components/Card/card.css]: use rem

File: Components/Card/card.css [hunk 4]
```diff
@@ -44,13 +50,18 @@
   grid-template-rows: 250px auto auto;
```
Comment (manish-GH) [inline Components/Card/card.css]: use rem

File: Components/Card/card.css [hunk 5]
```diff
@@ -44,13 +50,18 @@
...
 }
 .ver-lg-card {
+  width: 100%;
   max-width: 350px;
-  min-width: 100%;
   height: fit-content;
   display: grid;
   grid-template-rows: 200px auto;
```
Comment (manish-GH) [inline Components/Card/card.css]: use rem

**Review by manish-GH — APPROVED** [review]
Just replace px with rem wherever possible.
Your individual components look great. Nice work

**Comment by Shivani-1524** [comment]
> The code for the listed components looks good!
> 
> The horizontal card documentation seems a bit messed up you can address this in a later commit, looks good to merge for now! Screenshot 2022-02-13 at 7 00 30 PM

how did you get the card to look like this? no idea where i went wrong could you please dm me and let me know soon!

**Comment by shangkaul** [comment]
> > The code for the listed components looks good!
> > The horizontal card documentation seems a bit messed up you can address this in a later commit, looks good to merge for now! Screenshot 2022-02-13 at 7 00 30 PM
> 
> how did you get the card to look like this? no idea where i went wrong could you please dm me and let me know soon!

Shivani, I tried looking further. It seems the site looks fine on chrome, this issue crops up in safari. I still need to look into the css code to find the root of this issue. Probably one of the properties are not supported for safari OS. 

I wouldn't fret over this since this can be fixed by using a preprocessor on your CSS later.

File: Components/Button/button.css (hunk 1 above)
Comment (Shivani-1524) [reply Components/Button/button.css]: okay! will change it

File: Components/Card/card.css (hunk 2 above)
Comment (Shivani-1524) [reply Components/Card/card.css:22]: these units are too small for rem to have any significance

**Comment by Shivani-1524** [comment]
> Just replace px with rem wherever possible. Your individual components look great. Nice work

Okay will do that

**Comment by Shivani-1524** [comment]
> > > The code for the listed components looks good!
> > > The horizontal card documentation seems a bit messed up you can address this in a later commit, looks good to merge for now! Screenshot 2022-02-13 at 7 00 30 PM
> > 
> > 
> > how did you get the card to look like this? no idea where i went wrong could you please dm me and let me know soon!
> 
> Shivani, I tried looking further. It seems the site looks fine on chrome, this issue crops up in safari. I still need to look into the css code to find the root of this issue. Probably one of the properties are not supported for safari OS.
> 
> I wouldn't fret over this since this can be fixed by using a preprocessor on your CSS later.

oh okay thanks a lot for letting me know !! 😁 i will make sure to fix this later

File: Components/Card/card.css (hunk 4 above)
Comment (Shivani-1524) [reply Components/Card/card.css]: okay

File: Components/Card/card.css (hunk 5 above)
Comment (Shivani-1524) [reply Components/Card/card.css]: okay

File: Components/Card/card.css (hunk 2 above)
Comment (Shivani-1524) [reply Components/Card/card.css:22]: okay will do

File: Components/Button/button.css (hunk 1 above)
Comment (Shivani-1524) [reply Components/Button/button.css]: okay will do!!

**Pull request merged**